src/constructN.js

```diff
@@ -24,7 +24,7 @@ var nAry = require('./nAry');
  *      };
  *      Salad.prototype.recipe = function() {
  *        var instructions = R.map((ingredient) => (
- *          'Add a whollop of ' + ingredient, this.ingredients)
+ *          'Add a whollop of ' + ingredient), this.ingredients
```

Comment:
👍 for one line. 

House style is to use semicolons, so I am :+1: for that as well